test/integration/cdi/example-driver/app/server.go

```diff
@@ -249,9 +251,20 @@ func NewCommand() *cobra.Command {
 		Short: "run as kubelet plugin",
 		Long:  "cdi-example-driver kubelet-plugin runs as a device plugin for kubelet that supports dynamic resource allocation.",
 		Args:  cobra.ExactArgs(0),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			return fmt.Errorf("%s: not implemented", cmd.Use)
-		},
+	}
+	kubeletPluginFlagSets := cliflag.NamedFlagSets{}
+	fs = kubeletPluginFlagSets.FlagSet("kubelet")
+	pluginRegistrationPath := fs.String("plugin-registration-path", "/var/lib/kubelet/plugins_registry", "The directory where kubelet looks for plugin registration sockets, in the filesystem of the driver.")
+	endpoint := fs.String("endpoint", "/var/lib/kubelet/plugins/example-driver/dra.sock", "The Unix domain socket where the driver will listen for kubelet requests, in the filesystem of the driver.")
+	draAddress := fs.String("dra-address", "/var/lib/kubelet/plugins/example-driver/dra.sock", "The Unix domain socket that kubelet will connect to for dynamic resource allocation requests, in the filesystem of kubelet.")
```

Comment:
Can we have default values defined as constants?

Reply:
Would that increase readability?

As it stands now, everything required to understand a flag and the default is in one place.

If I define constants, I would have to duplicate the flag description as comment for the constant or leave it undocumented.

Reply:
Yes, it will increase readability in my opinion, but that's a matter of taste. If you prefer not to use constants, you should probably consider removing cdiVersion and deviceName constants as well.

Reply:
Yes. `cdiVersion` right now demonstrates what I described: the meaning of the constant is unclear unless one looks up how it is used. Comments can address that of course, but then one still has to read two lines that are very far away to verify that the usage matches the meaning.

Constants start to make sense when they get used often, but IMHO not when only used once.

Reply:
And `deviceName` cannot even *be* a constant, if my understanding is correct that it needs to be different for each claim...

Reply:
Done, see new latest commit.

Reply:
/lgtm